### ISSUE TITLE
Unable to install from Debian packagecloud.io repository

### ISSUE THREAD

**Issue by behrmann** [atom/atom#20409]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Steps to Reproduce

1. Install APT key and sources.list from https://packagecloud.io/AtomEditor/atom/install
2. `apt update`
3. Receive a warning about the repository missing a release file.

**Expected behavior:**

Not having to enable insecure repositories in apt.

**Actual behavior:**

This warning message from `apt update`
```
Ign:14 https://packagecloud.io/AtomEditor/atom/debian buster InRelease                         
Err:15 https://packagecloud.io/AtomEditor/atom/debian buster Release
  404  Not Found [IP: 130.133.8.114 80]
Reading package lists... Done
E: The repository 'https://packagecloud.io/AtomEditor/atom/debian buster Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Reproduces how often:**

100%

### Versions

Not really applicable.

### Additional Information

The last time this worked was February 7.

**Comment by lkashef** [comment]
Hey @behrmann, thanks for the report.

I know that this is a long shot since you said it was working around a week ago, any chance this helps https://askubuntu.com/questions/732985/force-update-from-unsigned-repository?

**Comment by behrmann** [comment]
@lkashef Yeah, this is not an option and I don't think this should be a recommended solution.

**Comment by lkashef** [comment]
@behrmann I am not recommending, I am trying to debug the issue, since it used to work before, so I am curious if it would work with this flag forced.

You also realize that linux distros are not signed, right?

**Comment by behrmann** [comment]
It doesn't seem to get rid of the message, neither does `--allow-insecure-repositories`, but even if it would, it would only get rid of the client side warning of the problem, but the problem should be with the release file in the repo.

> You also realize that linux distros are not signed, right?
I'm not quite sure what you mean by this, but in the specific case of APT repos the release file (containing hashes of packages) is signed. I am not aware of a major distro that does not have a package signing scheme.

**Comment by zafodB** [comment]
Hey @lkashef are there any updates on this? I'm still experiencing the problem as described in the original issue :)

**Comment by andericsilva** [comment]
same issue here. a workaround is the [manual installation](https://dev.to/andericsilva/como-instalar-o-atom-no-ubuntu-bionic-2hoo)

**Comment by RavanH** [comment]
Wow, I tried many LTS release repos and all from bionic up to focal but none have a Release file. This certainly explains why I've never seen an Atom update arrive :(

But I did find on https://unix.stackexchange.com/questions/515841/atom-editor-not-found-ubuntu-18-04 that manually setting the repo to any/main did the trick:
`deb [arch=amd64] https://packagecloud.io/AtomEditor/atom/any/ any main`

Too bad the installation script does not come to this solution on its own :/
